.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: java
+dist: bionic
+jdk: openjdk11
+env:
+  global:
+    - PYENV_VERSION=3.7
+    - AWS_REGION="us-east-1"
```

Comment:
why we need `AWS_REGION` and `AWS_DEFAULT_REGION`?

Reply:
I just lifted this from [here](https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-cloudformation/blob/master/.travis.yml)

Reply:
For this repo it's probably fine to just remove this bit actually.